request to merge cdl verification tool to master

anandkp92 opened the pull request.

mwetter reviewed (changes requested):
@anandkp92 : What is missing is a description of the file `realControllerPointMapping.json`. The documentation says there is such a file, but it does not describe what the file is. Can you please add this.

anandkp92 commented:
@mwetter I have added the file and added certain cross-references in the text as well. I wasn't able to preview the changes, so please let me know if the indentation or some text needs to change.
